hibernate-core/src/test/java/org/hibernate/test/locking/LockModeTest.java

```diff
@@ -201,6 +201,7 @@ public void testRefreshWithExplicitLowerLevelLockMode() {
 
 	@Test
 	@TestForIssue(jiraKey = "HHH-12257")
+	@RequiresDialectFeature(DialectChecks.SupportNoWait.class)
```

Comment:
can this be changed with @SkipForDialect( value = CockroachDB192Dialect.class

Reply:
done